src/Sylius/Bundle/ThemeBundle/Command/AssetsInstallCommand.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Sylius\Bundle\ThemeBundle\Command;
+
+use Sylius\Bundle\ThemeBundle\Asset\Installer\AssetsInstallerInterface;
+use Sylius\Bundle\ThemeBundle\Asset\Installer\OutputAwareInterface;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Command that places bundle web assets into a given directory.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
+ */
+class AssetsInstallCommand extends ContainerAwareCommand
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('assets:install')
+            ->setDefinition([
+                new InputArgument('target', InputArgument::OPTIONAL, 'The target directory', 'web'),
+            ])
+            ->addOption('symlink', null, InputOption::VALUE_NONE, 'Symlinks the assets instead of copying it')
+            ->addOption('relative', null, InputOption::VALUE_NONE, 'Make relative symlinks')
+            ->setDescription('Installs bundles web assets under a public web directory')
+            ->setHelp(<<<EOT
```

Comment:
Maybe return it from some private function, it's not needed to interfere readability in configuration method.